test/visual-diff/visual-tests

```diff
@@ -695,6 +695,22 @@
         ".i-amphtml-story-loaded",
       ],
     },
+    {
+      "url": "examples/visual-tests/amp-story/amp-story-interactive-img-poll.html",
+      "name": "amp-story-interactive-img-poll: basic test",
```

Comment:
Updating the name to a bit more descriptive. 
Rendering 4 images in mobile layout.